modules/ort/src/main/java/org/eclipse/sw360/antenna/ort/workflow/analyzers/OrtResultAnalyzer.java

```diff
@@ -10,22 +10,22 @@
  */
 package org.eclipse.sw360.antenna.ort.workflow.analyzers;
 
-import com.fasterxml.jackson.databind.JsonNode;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
+import com.here.ort.model.OrtResult;
```

Comment:
Nit pick: imports are not properly separated

Reply:
Correct, but AFAIR they weren't before either, so I was unsure about the policy. Will fix.